go-controller/pkg/util/util.go

```diff
@@ -139,18 +138,20 @@ func GetNodeAddresses(ipv4, ipv6 bool, nodes ...*v1.Node) (ipsv4 []net.IP, ipsv6
 		if err != nil {
 			return nil, nil, err
 		}
-		allips = allips.Insert(ips.UnsortedList()...)
+		allCIDRs = allCIDRs.Insert(ips.UnsortedList()...)
 	}
 
-	for _, ip := range allips.UnsortedList() {
-		ip := utilnet.ParseIPSloppy(ip)
+	for _, cidr := range allCIDRs.UnsortedList() {
+		ip, _, err := net.ParseCIDR(cidr)
+		if err != nil {
+			klog.Errorf("Failed to get host-addresses")
```

Comment:
Add `continue` here so we don't add `nil` ips to `ipsv4/ipsv6`

Reply:
done